e2e/Tests/iothub/device/FileUploadE2ETests.cs

```diff
@@ -26,85 +26,93 @@ public class FileUploadE2ETests : E2EMsTestBase
         private static readonly X509Certificate2 s_selfSignedCertificate = TestConfiguration.IotHub.GetCertificateWithPrivateKey();
 
         [TestMethod]
-        [TestCategory("LongRunning")]
-        [Timeout(LongRunningTestTimeoutMilliseconds)]
```

Comment:
These tests take a maximum of 2seconds to run, so updated their run cancelation time using cancellation tokens.